nservicebus/upgrades/3to4.md

```diff
@@ -146,6 +146,7 @@ NServiceBus PowerShell cmdlets have moved to NServiceBus.PowerShell.dll. See [Ma
 
 `configure.JsonSerializer()` has been obsoleted, please use `Configure.Serialization.Json();` instead. More info on supported serializers [here](/nservicebus/serialization/).
 
+The `XmlSerializer` will now automatically escape outgoing messages containing invalid characters. Version 3 endpoints will not be able to process those messages correctly.
```

Comment:
>Version 3 endpoints will not be able to process those messages correctly.

That reads scary, I think we need to qualify it. How about:

NOTE: This means that messages emitted by Version 4 endpoint containing invalid XML characters will not be processable by Version 3 endpoints